docs/core/testing/unit-testing-platform-extensions-output.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Microsoft.Testing.Platform output extensions
+description: Learn about the various Microsoft.Testing.Platform output extensions and how to use them.
+author: nohwnd
+ms.author: jajares
+ms.date: 08/26/2024
+---
+
+# Output extensions
+
+This article list and explains all `Microsoft Testing Platform` extensions related to the terminal output.
+
+## Terminal test reporter
+
+Terminal test reporter is the default implementation of status and progress reporting to the terminal (console).
+
+It comes built-in with Testing.Platform, and offers ANSI and non-ANSI mode, and progress indicator.
+
+### Output modes
+
+![Output with 1 failed test and a summary](./media/test-output-and-summary.png)
+There are two output modes available:
```

Comment:
I think it'd be good to have a sample of code and see the screenshot for each mode.

Reply:
added